RecoTracker/IterativeTracking/python/iterativeTkConfig.py

```diff
@@ -140,6 +140,12 @@ def createEarlySequence(eraName, postfix, modDict):
         seq += modDict[it]
     return seq
 
+def createEarlyTask(eraName, postfix, modDict):
```

Comment:
If there are additional changes to this PR, could you remove the above `createEarlySequence()` since it is not needed anymore? (otherwise I'll just do it in my next iterative tracking configuration cleanup round)

Reply:
I updated the branch with this change.

Reply:
Thanks.